tests/ui/attributes/key-value-expansion-scope.stderr

```diff
@@ -126,27 +126,6 @@ LL |     #![doc = in_block!()]
    |
    = help: have you added the `#[macro_use]` on the module/import?
 
-warning: cannot find macro `in_root` in the current scope when looking from the crate root
```

Comment:
I'd rather reorder some fields in AST to keep the old visiting order and keep the tests unchanged.
In some cases, like in macro scoping, the order is actually important, and it's a bit worrying to see it changing without seeing why.